package.json

```diff
@@ -28,30 +28,32 @@
     "scripts": {
         "setup": "./scripts/setup.js",
         "build": "./scripts/build-world.sh",
+        "build:legacy": "NODE_OPTIONS=--openssl-legacy-provider ./scripts/build-world.sh",
```

Comment:
do we need to keep a separate `:legacy` step in practice? What's the implication on consumers if we update the main steps with `NODE_OPTIONS=...` and make it clear we support node 20+ LTS (with a breaking change)

If anything, for me, it's the original ones that need a `:legacy` suffix since they're needed for legacy out of life versions of node

Reply:
This is just temporary and will be removed once we merge the storybook PR